.github/ISSUE_TEMPLATE/02-bug-report.yml

```diff
@@ -0,0 +1,28 @@
+name: 🐛 Bug report
+description: Something's not working
+title: 'fix: '
+labels: [bug]
+body:
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: 🐛 The bug
+      description: What isn't working? Describe what the bug is.
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: 🛠️ To reproduce
+      description: 'A reproduction of the bug. Please use the Nuxt OG Image starter https://stackblitz.com/edit/nuxt-starter-pkwfkx?file=pages%2Findex.vue'
+      placeholder: https://stackblitz.com/[...]
```

Comment:
#### Note

For the StackBlitz environment, I am using the same one as Nuxt Scripts.
(I don't think it's a high priority to create a dedicated environment, so I think it's fine to leave it as it is for now and handle it in a separate PR.)